.github/workflows/self-hosted.yml

```
@@ -12,8 +12,13 @@ jobs:
     - rpi4
     steps:
     - uses: actions/checkout@v3
```

```suggestion
    - uses: actions/checkout@v3
      with:
        submodules: recursive
```
Instead of separate `git submodule update --init --recursive` run (for both files)